Arrowgene.Ddon.GameServer/Tasks/HourlyTask.cs

```diff
@@ -0,0 +1,17 @@
+using Arrowgene.Ddon.Shared.Model.Scheduler;
+using System;
+
+namespace Arrowgene.Ddon.GameServer.Tasks
+{
+    public abstract class HourlyTask : SchedulerTask
+    {
+        public HourlyTask(TaskType type) : base(ScheduleInterval.Hourly, type)
+        {
+        }
+
+        public override long NextTimestamp()
+        {
+            return new DateTimeOffset(DateTime.Now.AddHours(1)).ToUnixTimeSeconds();
```

Comment:
I think I might need to be more careful with this. A long running server, this hour reset will eventually time drift. I might need to be more explicit and make it move to the next hour on the exact amount.

Reply:
Be careful with that, not in the hourly task, but in the daily/weekly ones, as hour changes such as daylight savings means that the time different wont be the exact same

Reply:
According to the MSDN, if we use `DateTime` using Now, it will account for DST based on that machines settings (needs automatic DST is enabled on that machine).

Reply:
This one is also interesting as it can be interpreted two ways. Does this mean "I always want to trigger this on the hour" or "I always want to trigger this after 60 minutes passes".

Reply:
With hours we're not going to have problems if we use AddHour or we add to the unix timestamp 60 * 60 * 1000. The problem is the daily task, which with DST it would be not 24 hours but 23/25 hours